src/app/screens/Dashboard/screens/Home/utils.js

```diff
@@ -0,0 +1,9 @@
+export const getComponentsCode = components =>
+  components?.length
+    ? components.map(comp => ({
```

Comment:
Can you add the type to `comp`? I think Apollo client should have the `@types` module